map/map.js

```diff
@@ -0,0 +1,11 @@
+const map = (array,callback)=> {
+    const newArray = []
+
+    for(let i = 0; i < array.length; i++){
+        newArray.push(callback(array[i]));
```

Comment:
This works great! For this exercise, we don't want to use array methods. Instead of `.push()` we want to find a way to append the updated value. Here you could set newArray on line 2 to a `let` and then reassign it on line 5, use spread and append the updated value `newArray = [...newArray, callback(array[i])]` :)